### PR TITLE
[MIG] website_event_questions_template: Migration to 12.0

### PR DESCRIPTION
**Pull request by victor-ditay**
cc @Tecnativa

### CODE REVIEW

**Comment by victor-ditay** [comment]
@sergio-teruel Please have a check

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
We used the trick of loading the questions in v10 with a button because there were no proper support for onchanges in nested one2many, but now I think the support is correct. Can you please switch back to fill things in onchange for not needing to save + press button?

**Comment by pedrobaeza** [comment]
Other minor thing is that a field with a number that I suppose is the sequence is shown when defining the question template.

**Comment by victor-ditay** [comment]
I will refactor based on your comments @pedrobaeza

**Comment by victor-ditay** [comment]
@pedrobaeza I pushed a new WIP commit so you could test if could work onchage option

**Comment by victor-ditay** [comment]
>Other minor thing is that a field with a number that I suppose is the sequence is shown when defining the question template.

@pedrobaeza I removed the previous commit, here is the reason https://github.com/odoo/odoo/issues/20163#issuecomment-505055793

File: website_event_questions_template/models/event_question_template.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# -*- coding: utf-8 -*-
-# Copyright 2017 Tecnativa - Sergio Teruel
+# Copyright 2017-2019 Tecnativa - Sergio Teruel
```
Comment (pedrobaeza) [inline website_event_questions_template/models/event_question_template.py]: Sergio hasn't done anything in 2019, so you shouldn't change its copyright. In any case, yours should be added in 2019, but any way, this file hasn't been modified, so you don't have to change whole copyright. This applies to all the files incorrectly modified.

File: website_event_questions_template/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+* Tecnativa (https://www.tecnativa.com)
+
+    * Sergio Teruel <sergio.teruel@tecnativa.com>
+
```
Comment (pedrobaeza) [inline website_event_questions_template/readme/CONTRIBUTORS.rst]: Add you as contributor

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by victor-ditay** [comment]
Changes done @pedrobaeza

File: website_event_questions_template/views/event_view.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<!-- Copyright 2017 Tecnativa - Sergio Teruel
+<!-- Copyright 2017-2019 Tecnativa - Sergio Teruel
```
Comment (pedrobaeza) [inline website_event_questions_template/views/event_view.xml]: This still remains

File: website_event_questions_template/views/website_event_questions_template_view.xml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<!-- Copyright 2017 Tecnativa - Sergio Teruel
+<!-- Copyright 2017-2019 Tecnativa - Sergio Teruel
```
Comment (pedrobaeza) [inline website_event_questions_template/views/website_event_questions_template_view.xml]: This still remains

**Comment by victor-ditay** [comment]
Changes done @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**